modular_zubbers/code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -0,0 +1,10 @@
+/datum/species/human/felinid
+	use_skintones = FALSE
+	species_traits = list(
```

Comment:
If you could add this:
`//Note that this replaces the list defined for /datum/species/human. If that list ever changes, this might need to be updated as well.`